README.md

```diff
@@ -68,21 +68,52 @@ In your starter code folder you will find every file you need to finish the game
+In the `api` folder, you need to create a `db.json` file. Inside our `db.json` we will specify the first 2 characters of our API, so we can start working with some data. Copy/paste the following characters in the file:
+
+```javascript
+{
+  "characters" : [
+    {
+      "name": "Han Solo",
+      "occupation": "Smuggler",
+      "debt": true,
+      "weapon": "Blaster Pistol",
+      "id": 1
+    },
+    {
+      "name": "Luke Skywalker",
+      "occupation": "Jedi Knight",
+      "debt": false,
+      "weapon": "Lightsaber",
+      "id": 2
+    }
+  ]
+}
+```
+
+Then run the following code on the terminal to make our API start working:
+
+```bash
+$ json-server --watch db.json --port 8000
+```
+
+### Iteration 2: The `APIHandler.js` file
+
+We have our API running, so now we will construct a class `APIHandler` to deal with the Axios calls. The only responsability of this class is to display the JSON result that comes from the API, or give the needed information to the API via a function argument.
 
 The funcionalities of the `APIHandler` class are:
 
-- Get all the characters info from *[http://ih-crud-api.herokuapp.com/characters](http://ih-crud-api.herokuapp.com/characters)*
-- Get a single character info from *[http://ih-crud-api.herokuapp.com/characters/:id](http://ih-crud-api.herokuapp.com/characters/:id)*
-- Create a single character posting the data to *[http://ih-crud-api.herokuapp.com/characters](http://ih-crud-api.herokuapp.com/characters)*
-- Delete a single character through his id in *[http://ih-crud-api.herokuapp.com/characters/:id](http://ih-crud-api.herokuapp.com/characters/:id)*
-- Edit a single character through his id in *[http://ih-crud-api.herokuapp.com/characters/:id](http://ih-crud-api.herokuapp.com/characters/:id)*
+- Get all the characters info from *[http://localhost:8000/characters](http://localhost:8000/characters)*
+- Get a single character info from *[http://localhost:8000/characters/:id](http://localhost:8000/characters/:id)*
+- Create a single character posting the data to *[http://localhost:8000/characters](http://localhost:8000/characters)*
+- Delete a single character through his id in *[http://localhost:8000/characters/:id](http://localhost:8000/characters/:id)*
+- Edit a single character through his id in *[http://localhost:8000/characters/:id](http://ih-crud-api.herokuapp.com/characters/:id)*
 
-You have to create an AJAX call for each of these actions. You can create as many functions as you need inside the class, but remember this class should only manage the API request and display the resulting value.
+You have to create an Axios call for each of these actions. You can create as many functions as you need inside the class, but remember this class should only manage the API request and display the resulting value.
```

Comment:
make them return a promise to super junior students could blow their mind because most of them do not even know what is a promise

Reply:
so, are we using promises without understanding them yet?
I'm not sure this is a good idea because we show them a bad practice (passing a callback to deal with asynchronous execution)